infra/ansible/playbooks/aggregator.yaml

```diff
@@ -0,0 +1,120 @@
+        src: '{{ ecdsa_keystore_path }}'
+        dest: /home/{{ ansible_user }}/.keystores/aggregator.ecdsa
+        owner: '{{ ansible_user }}'
+        group: '{{ ansible_user }}'
+
+    - name: Copy BLS keystore to server
+      copy:
+        src: '{{ bls_keystore_path }}'
+        dest: /home/{{ ansible_user }}/.keystores/aggregator.bls
+        owner: '{{ ansible_user }}'
+        group: '{{ ansible_user }}'
+
+    - name: Build aggregator
+      shell: 
+        chdir: /home/{{ ansible_user }}/repos/aggregator/aligned_layer/
+        cmd: /usr/local/go/bin/go build -o /home/{{ ansible_user }}/repos/aggregator/aligned_layer/build/aligned-aggregator /home/{{ ansible_user }}/repos/aggregator/aligned_layer/aggregator/cmd/main.go
+
+    - name: Upload config file for aggregator
+      template:
+        src: config-files/config-aggregator.yaml.j2
+        dest: "/home/{{ ansible_user }}/config/config-aggregator.yaml"
+      vars:
+        aligned_layer_deployment_config_file_path: "{{ lookup('ini', 'aligned_layer_deployment_config_file_path', file='ini/config-aggregator.ini') }}"
+        eigen_layer_deployment_config_file_path: "{{ lookup('ini', 'eigen_layer_deployment_config_file_path', file='ini/config-aggregator.ini') }}"
+        eth_rpc_url: "{{ lookup('ini', 'eth_rpc_url', file='ini/config-aggregator.ini') }}"
+        eth_rpc_url_fallback: "{{ lookup('ini', 'eth_rpc_url_fallback', file='ini/config-aggregator.ini') }}"
+        eth_ws_url: "{{ lookup('ini', 'eth_ws_url', file='ini/config-aggregator.ini') }}"
+        eth_ws_url_fallback: "{{ lookup('ini', 'eth_ws_url_fallback', file='ini/config-aggregator.ini') }}"
+        ecdsa_private_key_store_path: "{{ lookup('ini', 'ecdsa_private_key_store_path', file='ini/config-aggregator.ini') }}"
+        ecdsa_private_key_store_password: "{{ lookup('ini', 'ecdsa_private_key_store_password', file='ini/config-aggregator.ini') }}"
+        bls_private_key_store_path: "{{ lookup('ini', 'bls_private_key_store_path', file='ini/config-aggregator.ini') }}"
+        bls_private_key_store_password: "{{ lookup('ini', 'bls_private_key_store_password', file='ini/config-aggregator.ini') }}"
+        enable_metrics: "{{ lookup('ini', 'enable_metrics', file='ini/config-aggregator.ini') }}"
+        metrics_ip_port_address: "{{ lookup('ini', 'metrics_ip_port_address', file='ini/config-aggregator.ini') }}"
+
+    - name: Allow access to tcp port 8090
+      become: true
+      ufw:
+        rule: allow
+        port: 8090
+        proto: tcp
+      vars:
+        ansible_ssh_user: "{{ admin_user }}"
+
+    - name: Create systemd services directory
+      file:
+        path: "/home/{{ ansible_user }}/.config/systemd/user/"
+        state: directory
+
+    - name: Add service to systemd
+      template:
+        src: services/aggregator.service.j2
+        dest: "/home/{{ ansible_user }}/.config/systemd/user/aggregator.service"
+        force: no
+
+    - name: Start aggregator service
+      systemd_service:
```

Comment:
this should be: `ansible.builtin.systemd_service`

Reply:
Yeah, I'll migrate to FQCN (Fully Qualified Collection Name) for all tasks. I saw @JuArce's ansibles are written like that.

Reply:
I asked @JuArce whether we should use FQCN (Fully Qualified Collection Names) for all task names, or not. 

I'll update all task names if we use FQCN. If not, it should stay like that.

Reply:
We can move to a consistent convention on another PR